packages/next/build/webpack/plugins/font-stylesheet-gathering-plugin.ts

```diff
@@ -122,6 +122,10 @@ export class FontStylesheetGatheringPlugin {
           parser.hooks.call
             .for('__jsx')
             .tap(this.constructor.name, jsxNodeHandler)
+          // New React JSX transform:
+          parser.hooks.call
+            .for('imported var')
```

Comment:
What is this for? I believe there's support for this above.

Reply:
The new JSX transform adds an import, rather than declaring a variable, so the other code doesn’t automatically handle it (in webpack 4, at least)

Reply:
See https://github.com/vercel/next.js/pull/21046#issuecomment-759163371 for the test case that’s failing without this.